pup/utils/fact_extract.py

```diff
@@ -6,23 +6,25 @@
 from insights.util.subproc import CalledProcessError
 
 from insights.core.archives import InvalidArchive
-from insights.parsers.dmidecode import DMIDecode
 from insights.parsers.cpuinfo import CpuInfo
+from insights.parsers.dmidecode import DMIDecode
 from insights.parsers.installed_rpms import InstalledRpms
+from insights.parsers.ip import IpAddr
 from insights.parsers.lsmod import LsMod
 from insights.parsers.meminfo import MemInfo
+from insights.parsers.ps import PsAuxcww
 from insights.parsers.redhat_release import RedhatRelease
-from insights.parsers.uname import Uname
 from insights.parsers.systemd.unitfiles import UnitFiles
+from insights.parsers.uname import Uname
 from insights.parsers.virt_what import VirtWhat
 from insights.specs import Specs
-from insights.util.canonical_facts import get_canonical_facts, IPs
+from insights.util.canonical_facts import get_canonical_facts
```

Comment:
apologies for sorting these in this PR, it makes the changes harder to read :eyeglasses: 

`PsAuxcww` and `IPAddr` were added, and `IPs` was removed

Reply:
no worries!